src/Fmt_ast.ml

```diff
@@ -344,6 +344,10 @@ let rec fmt_longident (li: Longident.t) =
   | Lapply (li1, li2) ->
       cbox 0 (fmt_longident li1 $ wrap "(" ")" (fmt_longident li2))
 
+let fmt_longident_loc c ?(pre= Caml.format_of_string "")
```

Comment:
is this preferable to using `("" : _ format)` ?

Reply:
fixed